### DISCUSSION TITLE
All my feeds broke due to unstable connection

### DISCUSSION THREAD

**Discussion by nikhilCad** [FreshRSS/FreshRSS#6309] [Q&A] [answered]
~~My internet was unstable. I started the feed reload and internet went out halfway through, when this reload completed ( with no internet), ALL of my feed broke~~
![image](https://github.com/FreshRSS/FreshRSS/assets/47214394/ef031019-d377-409b-bd99-19cc0156e2ba)

~~Literally every feed, is there any way to reset this?~~
![image](https://github.com/FreshRSS/FreshRSS/assets/47214394/dfc2ad74-2cf0-4fc3-9a67-6430f1113d89)
~~My setup -> Running on Windows 10 using xampp (no docker as it slows down my pc). I have been using freshrss for a like a week and only now I am getting this issue.~~

Edit: Weird, I was refreshing it before and it said something like "No feeds to refresh". Then I waited for around 10-15 minutes and it refreshed, and all feeds are back. Fixed, thanks

**Comment by Alkarex** [comment, marked as answer]
Yes, there is a cache to avoid too frequent refreshes (which leads to other problems such as being banned etc.) although we could try to be smarter at detecting the type of error, and how long it should be cashed
